plugins/polio/api/notifications/serializers.py

```diff
@@ -0,0 +1,48 @@
+from rest_framework import serializers
+
+from plugins.polio.models import Notification, NotificationImport
+
+
+class NotificationImportSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = NotificationImport
+        fields = ["id", "account", "file", "created_by"]
+        extra_kwargs = {
+            "id": {"read_only": True},
+        }
+
+
+class NotificationSerializer(serializers.ModelSerializer):
+    district = serializers.CharField(source="annotated_district", read_only=True)
+    province = serializers.CharField(source="annotated_province", read_only=True)
+    country = serializers.CharField(source="annotated_country", read_only=True)
+
+    class Meta:
+        model = Notification
+        fields = [
+            "epid_number",
+            "vdpv_category",
+            "source",
+            "vdpv_nucleotide_diff_sabin2",
+            "lineage",
+            "closest_match_vdpv2",
+            "date_of_onset",
+            "date_results_received",
+            "district",
+            "province",
+            "country",
+            "site_name",
+            "created_at",
+            "updated_at",
+            # Write only.
```

Comment:
That's read only, no?

Reply:
Fixed! The backend is now setting `account` and `user` related fields https://github.com/BLSQ/iaso/pull/928/commits/a014b52a7f0a02756afdd7fdd7c6f3cf51f073e9